pkg/api/v20210131preview/openshiftcluster_validatestatic.go

```diff
@@ -210,6 +211,14 @@ func (sv *openShiftClusterStaticValidator) validateNetworkProfile(path string, n
 		}
 	}
 
+	switch np.SDNProvider {
+	case SDNProviderOVNKubernetes, SDNProviderOpenShiftSDN:
+	default:
+		fmt.Println(np.SDNProvider)
+		errorMsg := fmt.Sprintf("The provided SDNProvider must be either '%s' or '%s'.", SDNProviderOVNKubernetes, SDNProviderOpenShiftSDN)
```

Comment:
`The provided SDNProvider '%s' is invalid.", value`
We dont provide enum values in the error messages. This is not good practice to leak API types into error static messages as it makes user side parsing complicated.

Reply:
Updated.